### PR TITLE
jootc.com - Anti-RightClick

### PR DESCRIPTION
**Pull request by piquark6046**
##### To avoid invalid pull requests, please check and confirm following checkboxes:

  - [X] This is not an ad/bug report;
  - [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
  - [X] I have performed a self-review of my own changes;
  - [X] My changes do not break web sites, apps and files structure.

##### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

  - [ ] Missed ads or ad leftovers;
  - [ ] Website or app doesn't work properly;
  - [ ] AdGuard gets detected on a website;
  - [ ] Missed analytics or tracker;
  - [ ] Social media buttons — share, like, tweet, etc;
  - [X] Annoyances — pop-ups, cookie warnings, etc;
  - [x] Filters maintenance.

##### Enter the issue address:

- https://github.com/AdguardTeam/AdguardFilters/issues/119143

### Terms

  - [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met.

### CODE REVIEW

File: AnnoyancesFilter/sections/tweaks.txt [hunk 1]
```diff
@@ -1885,7 +1885,12 @@ legionscans.com,a2zapk.com,flinsetyadi.com,theaircurrent.com,fully-fundedscholar
 now.rememberapp.co.kr,melodelaa.link,asus-zenfone.com,techieway.blogspot.com,chronologia.pl#%#//scriptlet("abort-current-inline-script", "document.onselectstart")
 ! #%#//scriptlet("abort-current-inline-script", "$", "contextmenu")
 hinfomax.co.kr,britg.kr,japanxxxmovie.com,sexpox.com,47news.jp,learn-from-homey.blogspot.com,belajarbro.id#%#//scriptlet("abort-current-inline-script", "$", "contextmenu")
+! #%#//scriptlet("set-constant", "disable_copy", "noopFunc")
+jootc.com,dora-guide.com,codedosa.com,gradium.co.kr,semesters.in#%#//scriptlet("set-constant", "disable_copy", "noopFunc")
+! #%#//scriptlet("set-constant", "disable_copy_ie", "noopFunc")
+jootc.com,dora-guide.com,codedosa.com,gradium.co.kr,semesters.in#%#//scriptlet("set-constant", "disable_copy_ie", "noopFunc")
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/tweaks.txt:1891]: Better change existing rules
```
dora-guide.com,codedosa.com,gradium.co.kr,semesters.in#%#//scriptlet('set-constant', 'disable_copy', 'noopFunc')
dora-guide.com,codedosa.com,gradium.co.kr,semesters.in#%#//scriptlet('set-constant', 'disable_copy_ie', 'noopFunc')
```

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/tweaks.txt (hunk 1 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/tweaks.txt:1891]: Also I see another rules for these domains in this file. Can be removed?

Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt]: Checked.

File: AnnoyancesFilter/sections/tweaks.txt [hunk 2]
```diff
@@ -1883,7 +1883,12 @@ legionscans.com,a2zapk.com,flinsetyadi.com,theaircurrent.com,fully-fundedscholar
 now.rememberapp.co.kr,melodelaa.link,asus-zenfone.com,techieway.blogspot.com,chronologia.pl#%#//scriptlet('abort-current-inline-script', 'document.onselectstart')
 ! #%#//scriptlet('abort-current-inline-script', '$', 'contextmenu')
 hinfomax.co.kr,britg.kr,japanxxxmovie.com,sexpox.com,47news.jp,learn-from-homey.blogspot.com,belajarbro.id#%#//scriptlet('abort-current-inline-script', '$', 'contextmenu')
+! #%#//scriptlet("set-constant", 'disable_copy', 'noopFunc')
+jootc.com,dora-guide.com,codedosa.com,gradium.co.kr,semesters.in#%#//scriptlet("set-constant", "disable_copy", "noopFunc")
+! #%#//scriptlet("set-constant", 'disable_copy_ie', 'noopFunc')
+jootc.com,dora-guide.com,codedosa.com,gradium.co.kr,semesters.in#%#//scriptlet("set-constant", "disable_copy_ie", "noopFunc")
```
[the remark this answers is not in the document]
Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt:1889]: Okay. I just remove the old rules.

**Review by Alex-302 — APPROVED** [review]

File: AnnoyancesFilter/sections/tweaks.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Alex-302) [reply AnnoyancesFilter/sections/tweaks.txt:1889]: Thanks.

**Pull request merged**